src/main/java/castalia/Dist.java

```diff
@@ -0,0 +1,381 @@
+ package castalia;
+
+ /**
+  * From: http://www.stat.ubc.ca/~ruben/ILAS_2009/ilas_applets/continuous/Dist.java
+  */
+ public final class Dist
+{
+    //No Constructor
+//Constants
+    public static final double SQRT2PI=Math.sqrt(6.283185307179586);
+    public static final double DBL_EPSILON=2.2204460492503131e-16;
+    public static final double M_LN_SQRT_2PI= 0.918938533204672741780329736406;
+//Class methods
+
+    /** Standard normal density */
+    public static double dnorms(double z)
+    {  double pdf= Math.exp(-z * z / 2.0);
+        pdf /= SQRT2PI;
+        return pdf;
+    }
+
+    /** standard normal cdf */
+    public static double pnorms(double x)
+    {
+        double c[]= {
+                0.39894151208813466764, 8.8831497943883759412,
+                93.506656132177855979, 597.27027639480026226,
+                2494.5375852903726711, 6848.1904505362823326,
+                11602.651437647350124, 9842.7148383839780218,
+                1.0765576773720192317e-8 };
+
+        double d[]= {
+                22.266688044328115691, 235.38790178262499861,
+                1519.377599407554805, 6485.558298266760755,
+                18615.571640885098091, 34900.952721145977266,
+                38912.003286093271411, 19685.429676859990727 };
+
+        double p[] = {
+                0.21589853405795699, 0.1274011611602473639,
+                0.022235277870649807, 0.001421619193227893466,
+                2.9112874951168792e-5, 0.02307344176494017303 };
+
+        double q[] = {
+                1.28426009614491121, 0.468238212480865118,
+                0.0659881378689285515, 0.00378239633202758244,
+                7.29751555083966205e-5 };
+
+        double a[] = {
+                2.2352520354606839287, 161.02823106855587881,
+                1067.6894854603709582, 18154.981253343561249,
+                0.065682337918207449113 };
+
+        double b[] = {
+                47.20258190468824187, 976.09855173777669322,
+                10260.932208618978205, 45507.789335026729956} ;
+
+        double one = 1.0;
+        double half = 0.5;
+        double zero = 0.0;
+        double sixten = 1.6;
+        double sqrpi = 1./SQRT2PI;
+        double thrsh = 0.66291;
+        double root32 = 5.6568542494923806;
+        final double DBL_EPSILON=2.2204460492503131e-16;
+        //final double DBL_MIN=2.2250738585072014e-308; ?
+
+        double xden, temp, xnum, result, ccum;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://www.codacy.com/app/scala-academy/Castalia/file/1968742700/issues/source?bid=2998213&fileBranchId=3112713#l67)